templates/master/00-master/on-prem/files/keepalived-keepalived.yaml

```diff
@@ -37,7 +37,7 @@ contents:
     # TODO: Improve this check. The port is assumed to be alive.
     # Need to assess what is the ramification if the port is not there.
     vrrp_script chk_ingress {
-        script "/usr/bin/timeout 0.9 /usr/bin/curl -o /dev/null -Lfs http://localhost:1936/healthz/ready"
+        script "/usr/bin/timeout 0.9 /usr/bin/curl -o /dev/null --max-time 1 -Lfs http://localhost:1936/healthz/ready"
```

Comment:
Isn't this redundant, as timeout will kill the the curl before 1 second in any case. Any explicit cleanup curl does after 1 seconds would presumably not run.

Reply:
Ah, you remove the timeout in the next commit anyway. If you update probably best to elide this commit.

Reply:
NP haha. Yeah. I'm still working on proving this out, but my hypothesis right now is that syskilling the curl rather than using the built in timeout may not be closing the tcp connection. Yet to prove that though.